FakeNFT/Scenes /Profile/ProfileInteractor.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ProfileInteractor.swift
+//  FakeNFT
+//
+//  Created by Alibi Mailan on 24.02.2025
+//
+
+import Foundation
+
+protocol ProfileInteractorProtocol: AnyObject {
+    func fetchProfile(userId: String)
+}
+
+final class ProfileInteractor: ProfileInteractorProtocol {
+    weak var presenter: ProfilePresenterProtocol?
+    
+    // MARK: - Private
+    private let profileService: ProfileService
+    
+    init(profileService: ProfileService) {
+        self.profileService = profileService
+    }
+    
+    func fetchProfile(userId: String) {
+        profileService.loadProfile(id: userId) { (result) in
```

Comment:
с одним аргументом можно и без скобочек, и swiftlint будет ругаться на это на мердже
```suggestion
        profileService.loadProfile(id: userId) { result in
```